internal/services/sentinel/sentinel_alert_rule_fusion_resource.go

```diff
@@ -62,6 +62,62 @@ func resourceSentinelAlertRuleFusion() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"source_setting": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				// Service will auto-fill this if not given in request, based on the "alert_rule_template_guid".
+				Computed: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"name": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
+						"source_sub_type": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Resource{
+								Schema: map[string]*pluginsdk.Schema{
+									"name": {
+										Type:         pluginsdk.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringIsNotEmpty,
+									},
+									"enabled": {
+										Type:     pluginsdk.TypeBool,
+										Optional: true,
+										Default:  true,
+									},
+									"enabled_severities": {
```

Comment:
should this be
```suggestion
									"severities_enabled": {
```

Reply:
@katbyte We used `foo_enabled` as a boolean property, whilst this one is an array of enabled severities.

Reply:
yes but putting it in front doesn't make sense. looking at the api closer, should this just be
```suggestion
									"severities": {
```
as it is the severities for this sub_type, so `sub_type.enable = true` means enable then, `sub_type.enabled = false` would actually disable them?

so maybe
```suggestion
									"severities_allowed": {
```
or
```suggestion
									"severities_filter": {
```

Reply:
@katbyte I've renamed it to `severities_allowed` now.